variable/technology/technologies.yaml

```
@@ -460,3 +531,326 @@ Investment|Energy Supply|Liquids|Oil:
                 capturing equipment should be included but not the one on
                 transport and storage.
    unit: billion US$2010/yr
+
+Line Type|Electricity|Transmission|Line:
+   description: Line type: alternating current (AC) and direct current (DC)
+   unit: [AC,DC]
```

for python consistency...
```suggestion
   unit: [AC, DC]
```